src/lib.rs

```diff
@@ -2,19 +2,20 @@
 //!
 //! # Example
 //! ```rust,no_run
-//! use hyper::{Client, Request, Uri};
+//! use hyper::{Client, Request, Uri, body::HttpBody};
 //! use hyper::client::HttpConnector;
 //! use futures::{TryFutureExt, TryStreamExt};
```

Comment:
These types are no longer required. Also, the README.md example should be updated.